src/wabt.cpp

```diff
@@ -104,7 +107,15 @@ ExecutionResult WabtEngine::execute(
     ) {
       int64_t gas = static_cast<int64_t>(args[0].value.i64);
       // FIXME: handle host trap here
-      interface.eeiUseGas(gas);
+      try {
+        interface.eeiUseGas(gas);
+      } catch (HeraException const& e) {
```

Comment:
There's no need to name these arguments.

Reply:
Ok.